src/alf/atoms.ts

```diff
@@ -176,43 +176,59 @@ export const atoms = {
   },
   text_2xs: {
     fontSize: tokens.fontSize._2xs,
+    letterSpacing: 0.25,
```

Comment:
All our existing type styles have built in letter spacing, so for consistency I've added here too